module_auto_update/models/module.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+import logging
+
+from odoo import api, fields, models
+from odoo.modules.module import get_module_path
+
+_logger = logging.getLogger(__name__)
+try:
+    from checksumdir import dirhash
+except ImportError:
+    _logger.debug('Cannot `import checksumdir`.')
+
+
+class Module(models.Model):
+    _inherit = 'ir.module.module'
+
+    checksum_dir = fields.Char(
+        compute='_compute_checksum_dir',
+    )
+    checksum_installed = fields.Char()
```

Comment:
I feel this one should be read only, mostly now that you are exposing it in the UI.
You should also add `string` and `help` attributes to both fields for the same reason, to let the user know what's this.